Fix description (#388)
contao/docs: docs/dev/reference/hooks/validateFormField.md

    Object of the current front end widget. Use it to access form field properties.

2. *string* `$formId`

>     Alias of the current form. Used in the `value` attribute of the hidden form field `FORM_SUBMIT`. Don’t confuse with `$form->id`.

Reviewer: ```suggestion
    Alias of the current form. Used in the `value` attribute of the hidden form field `FORM_SUBMIT`. Do not confuse with `$form->id`.
```
